Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -159,8 +159,11 @@ var DrawerLayoutAndroid = React.createClass({
 
   render: function() {
     var drawStatusBar = Platform.Version >= 21 && this.props.statusBarBackgroundColor;
+    var drawerSubviewBackgroundColor = (this.props.style || {}).backgroundColor || 'white';
```

Comment:
Will this not break if something is passing `style={styles.drawerBackgroundColor}` where `drawerbackgroundColor` is a number?

we can explicitly use a prop for the backgroundColor something like contentBackgroundColor or implement this like `scrollview's` `contentContainerStyle`